<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="0" failed="0" skipped="4">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-23T18:16:01 IST" name="Default suite" finished-at="2022-12-23T18:16:01 IST" duration-ms="43">
    <groups>
    </groups>
    <test started-at="2022-12-23T18:16:01 IST" name="Default test" finished-at="2022-12-23T18:16:01 IST" duration-ms="43">
      <class name="testing.dev.BuyerPortalLogin">
        <test-method is-config="true" signature="setUp()[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="setUp" finished-at="2022-12-23T18:16:01 IST" duration-ms="6" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException

at testing.dev.BaseTest.setUp(BaseTest.java:112)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="setReports()[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="setReports" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561470" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setReports -->
        <test-method is-config="true" signature="testName(java.lang.reflect.Method)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="testName" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561489" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testName -->
        <test-method signature="signInPageTest4()[pri:1, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="signInPageTest4" finished-at="2022-12-23T18:16:01 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException

at testing.dev.BaseTest.setUp(BaseTest.java:112)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInPageTest4 -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="tearDown" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561493" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="signUpPageTest1()[pri:1, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="signUpPageTest1" finished-at="2022-12-23T18:16:01 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException

at testing.dev.BaseTest.setUp(BaseTest.java:112)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUpPageTest1 -->
        <test-method is-config="true" signature="testName(java.lang.reflect.Method)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="testName" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561497" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testName -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="tearDown" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561500" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="testName(java.lang.reflect.Method)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="testName" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561501" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testName -->
        <test-method signature="signUpPageTest2()[pri:2, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="signUpPageTest2" finished-at="2022-12-23T18:16:01 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException

at testing.dev.BaseTest.setUp(BaseTest.java:112)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUpPageTest2 -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="tearDown" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561504" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="testName(java.lang.reflect.Method)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="testName" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561506" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testName -->
        <test-method signature="signInPageTest3()[pri:3, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="signInPageTest3" finished-at="2022-12-23T18:16:01 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Unhandled exception type IOException
Unhandled exception type IOException

at testing.dev.BaseTest.setUp(BaseTest.java:112)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInPageTest3 -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="tearDown" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561509" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="closeReport()[pri:0, instance:testing.dev.BuyerPortalLogin@f5ac9e4]" started-at="2022-12-23T18:16:01 IST" name="closeReport" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1671799561510" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeReport -->
      </class> <!-- testing.dev.BuyerPortalLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
